devtools/conda-envs/test.yml

```diff
@@ -2,22 +2,15 @@ name: alchemiscale-test
 channels:
   - jaimergp/label/unsupported-cudatoolkit-shim
   - conda-forge
-  - openeye
 dependencies:
   - pip
 
-  # gufe dependencies
-  - numpy<1.24  # https://github.com/numba/numba/issues/8615#issuecomment-1360792615
-  - networkx
-  - rdkit
+  # alchemiscale dependencies
+  - gufe
+  - openfe
```

Comment:
not blocking but doing `>=` for min versions we support would save us from pulling in something old by mistake